.github/workflows/submitty_ci.yml

```diff
@@ -547,7 +399,76 @@ jobs:
           - name: Restart autograding worker
             run: |
               sudo systemctl restart submitty_autograding_worker
+    e2eCypressTwo:
```

Comment:
I don't think there's a need to have two separate Cypress jobs here.  All of the stuff in this job takes less than 5 minutes to complete, based on the most recent CI runs.  The setup will likely take longer than the actual testing here.